### PR TITLE
Fix nameservice tests

### PR DESCRIPTION
**Pull request by bufrr**
### Proposed changes in this pull request
Explain the changes in this pull request in order to help the project maintainers understand the overall impact of it.

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [ ] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [ ] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [ ] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

**Comment by bufrr** [comment]
fix #891 #892

**Comment by abebeos** [comment]
@bufrr , very nice, I've update my PR #893 (@yilunzhang , maybe review this one too while you're at it).

File: config/heightdepvar.go [hunk 1]
```diff
@@ -45,14 +45,14 @@ func (hdi *HeightDependentUint256) GetValueAtHeight(height uint32) common.Uint25
 }
 
 type HeightDependentBool struct {
-	heights []uint32
-	values  []bool
+	Heights []uint32
```
Comment (yilunzhang) [inline config/heightdepvar.go:48]: It's better to change all `HeightDependentXXX` variables to make them consistent

File: .gitignore [hunk 2]
```diff
@@ -20,6 +20,8 @@ cscope*
 /Log/
 /vendor/
 /test/
+chain/store/TestChainDB
```
Comment (yilunzhang) [inline .gitignore:23]: There can be added to file `chain/store/.gitignore`

**Review by yilunzhang — APPROVED** [review]
Looks good to me

**Pull request merged**